packages/flutter/lib/src/painting/image_cache.dart

```diff
@@ -577,7 +587,18 @@ class _CachedImage {
   _CachedImage(this.completer, this.sizeBytes);
 
   final ImageStreamCompleter completer;
-  final int sizeBytes;
+  int sizeBytes;
+}
+
+class _LiveImage extends _CachedImage {
+  _LiveImage(ImageStreamCompleter completer, int sizeBytes, this.remover)
+      : super(completer, sizeBytes);
+
+  final VoidCallback remover;
```

Comment:
nit: violates style guide for naming of callbacks.

Reply:
chagning to handleRemove